### ISSUE TITLE
SideMenu Clicking outside the panel [bug]

### ISSUE THREAD

**Issue by sabuto** [kikipoulet/SukiUI#5]
When you click outside of the side menu on the main page, the menu retracts to the smaller size like it's supposed to but doesn't change to the smaller menu when you click the arrow (so just the icons).

**Comment by kikipoulet** [comment]
Yep that's weird, we have to see if there is a problem with the IsPaneOpen property not updating or something like that
